lua/lualine/components/branch.lua

```diff
@@ -8,12 +8,22 @@ local get_git_branch = async:new({
     if data then
       git_branch = data:gsub('\n', '')
     end
-  end
+  end,
+  on_stderr = function (_, data)
+    if data then
+      if data:find("fatal: not a git repository") then
+        git_branch = ''
+      end
+    end
+  end,
 })
 
 local timer = vim.loop.new_timer()
 timer:start(0, 1000, vim.schedule_wrap(function()
+  local cur_dir = vim.fn.getcwd()
+  vim.api.nvim_set_current_dir(vim.fn.expand("%:p:h"))
```

Comment:
This should be error checked, right now it produces errors in `fzf` in floating mode, probably due to temp buffers or sth.
![pic-window-210108-1015-58](https://user-images.githubusercontent.com/41551030/103996745-93b72980-519a-11eb-8c68-dacf67b9677b.png)

Reply:
Okay I resolved this, should be working now.